### PR TITLE
[Merged by Bors] - Support offline mode

### PR DESCRIPTION
**Pull request by maltesander**
# Description

- Using new opa images that contain the bundle builder
- Changed init and bundle builder container to use the opa image

fixes https://github.com/stackabletech/opa-operator/issues/389

## Review Checklist

- [ ] Code contains useful comments
- [ ] CRD change approved (or not applicable)
- [ ] (Integration-)Test cases added (or not applicable)
- [ ] Documentation added (or not applicable)
- [ ] Changelog updated (or not applicable)
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
- [ ] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,11 +10,13 @@ All notable changes to this project will be documented in this file.
 - `operator-rs` `0.22.0` -> `0.27.1` ([#377]).
 - Don't run init container as root and avoid chmod and chowning ([#382]).
 - [BREAKING] Use Product image selection instead of version. `spec.version` has been replaced by `spec.image` ([#385]).
+- Updated to new docker tags containing the opa-bundle builder ([#391]).
```
Comment (sbernauer) [inline CHANGELOG.md:13]:
```suggestion
- Support offline mode ([#391]).
- Updated to new docker tags containing the opa-bundle builder ([#391]).
```

File: docs/modules/ROOT/pages/implementation-notes.adoc [hunk 2]
```diff
@@ -14,13 +14,13 @@ We ensure local access via an https://kubernetes.io/docs/concepts/services-netwo
 
 Users can manage policy rules by creating, updating and deleting `ConfigMap` resources.
 
-The role of the bundle builder is to convert these resources to bundles (`tar.gz` files) and make them available as an HTTP endpoint.
-The bundle builder runs in a side container of the OPA `Pod` as a simple HTTP server that OPA is querying regularly
+The role of the https://github.com/stackabletech/opa-bundle-builder[OPA Bundle Builder] is to convert these resources to bundles (`tar.gz` files) and make them available as an HTTP endpoint.
```
Comment (sbernauer) [inline docs/modules/ROOT/pages/implementation-notes.adoc:17]: Role might be confusing ;)
```suggestion
The responsibility of the https://github.com/stackabletech/opa-bundle-builder[OPA Bundle Builder] is to convert these resources to bundles (`tar.gz` files) and make them available via an HTTP endpoint.
```

**Review by sbernauer — APPROVED** [review]
lgtm when test pass

**Comment by maltesander** [comment]
test https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/opa-operator-it-custom/27/

**Pull request closed without merging**